testdata/rabbit/write2disk/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"os"
+	"strings"
+)
+
+func main() {
+	message := "Go is expressive, concise, clean, and efficient. " +
+		"Its concurrency mechanisms make it easy to write programs" +
+		"that get the most out of multicore and networked machines, " +
+		"while its novel type system enables flexible and modular program construction. " +
+		"Go compiles quickly to machine code yet has the convenience of garbage" +
+		"collection and the power of run-time reflection."
+	// change the repeat count value to write longer data
+	longString := strings.Repeat(message, 1000000)
```

Comment:
I doubt that 1000000 here is a problem.... *BUT* I would probably use something smaller like 1000.

The objective is test what happens when we run out of disk.... and it is at least possible that 1M copies of that string will run out of MEMORY before it tests the disk.